chapterone/HelloWorld.java

```diff
@@ -0,0 +1,8 @@
+public class HelloWorld
+{
+    public static void main(String [] args)
+    {
+        System.out.print("Hello, World");
```

Comment:
This is very classic. Have you thought about spicing it up or saying hello in a different language?